students/Preethi_Krishnan/lesson_03/mail_room_1.py

```diff
@@ -0,0 +1,62 @@
+
+donate_list = [['William Gates', 500, 500.5], ['Mark Zuckerberg', 300, 300.6, 500, 600], ['Jeff Bezos', 800, 970.44], ['Paul Allen', 1, 1000, 780.5]]
+
+
+def donor_names():
+    donor_name = []
+    for name in donate_list:
+        donor_name.append(name[0])
+    return donor_name
+
+
+def thank_you_note():
+    donor_name = raw_input("Please enter the Full Name of the donor to send thank you note or enter \"list donors\" to see the list of donors:")
+    if donor_name == '0':
+        return
+    else:
+        if donor_name == 'list donors':
+            print("List of Donors are:{}".format(donor_names()))
+            thank_you_note()
+        elif donor_name not in donor_names():
```

Comment:
Instead of calling `donor_names()` twice, set it to a variable above the first `if` here.